saleor/graphql/discount/tests/test_pagination.py

```diff
@@ -5,47 +5,49 @@
 from freezegun import freeze_time
 
 from ....discount import DiscountValueType, VoucherType
-from ....discount.models import Sale, Voucher
+from ....discount.models import Sale, SaleChannelListing, Voucher
 from ...tests.utils import get_graphql_content
 
 
 @pytest.fixture
 @freeze_time("2020-03-18 12:00:00")
-def sales_for_pagination(db):
+def sales_for_pagination(db, channel_USD):
```

Comment:
If we add fixture witch use db we can delete db form params. 
```suggestion
def sales_for_pagination(channel_USD):
```